databricks/koalas/plot.py

```diff
@@ -265,6 +268,11 @@ def update_dict(dictionary, rc_name, properties):
                             if ci[1] is not None:
                                 stats["cihi"] = ci[1]
 
+        if LooseVersion(matplotlib.__version__) < LooseVersion("3.1.0"):
+            extra_args = {"manage_xticks": bool(manage_ticks or manage_xticks)}
+        else:
+            extra_args = {"manage_ticks": bool(manage_ticks or manage_xticks)}
```

Comment:
We could make it more fine-grained like considering precedence e.g.) `False` and `True` -> `False`. But I think it's too much ..